PROJ-19 Architecture of the game (#11)
SW805F20/report: sections/sprint-1/architecture.tex

\section{Architecture of the game}
In this section we will look at the overall architecure of the different components involved in the game.
\begin{figure}[H]
    \centering
    \includegraphics[width=0.6\linewidth]{Architecture.png}
    \caption{The different components of the game}
    \label{fig:architecture}
\end{figure}

> In \autoref{fig:architecture} an illustration of how we imagine the different components of the game are going to work together is shown.

Reviewer: ```suggestion
\noindent
In \autoref{fig:architecture} an illustration of how we imagine the different components of the game are going to work together is shown.
```

The arrows in the diagram show the flow of data in the game.
The Pozyx component is responsible for providing the positional data to the host. 

> Each client/player is equipped with one of the tags which tracks their position on the field. 

Reviewer: ```suggestion
Each client/player is equipped with one of the tags that track their positions on the field. 
```

> The playing field's dimensions will be set to match with the distances between the anchors. 

Reviewer: ```suggestion
The playing field's dimensions will be set to match the distances between the anchors. 
```

The host has the positional data for each client, but it will also need to know which of the tags each client is equipped with.
The host will then continuosly transfer positional data to the clients.

> This information will also include which of the tag ids that belong to each clients.

Reviewer: ```suggestion
This information will also include which of the tag ids belong to which clients.
```
